README.md

```diff
@@ -1,11 +1,31 @@
-## universal_card_catalog
+# universal_card_catalog
 
 A basic card catalog for any gaming system.  The goal for this sucker is to assist in a closed source game project I'm working on.  While the game might be closed source, this card catalog will be universal.  Works with D&D, RTS games, and any other system with monsters, units, or heroes.
 
 ====
 
-#### What?
+## What?
 - create CSV `pipe separated` data files for your favorite systems
 - modeled after the rank/paragon unit system made popular by **Warcraft III** and **Starcraft:BW**
 - add animated portrait and unit `pngs`, `jpegs`, and [gifs](http://vignette2.wikia.nocookie.net/starcraft/images/e/ed/Marine_SC2_GameAnim1.gif/revision/latest?cb=20080605172207).
 - run as many analytics and diagnostics as you prefer on their databanks.  Min-Max~!
+
+
+## Run & Build Command Examples
+
+`npm run build` will compile the whole site.
```

Comment:
Before `npm run build` will work, run `npm install`

After that you can `npm start`